android/java/org/chromium/chrome/browser/rewards/onboarding/RewardsOnBoardingAllSet.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package org.chromium.chrome.browser.rewards.onboarding;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.os.Build;
+import android.view.LayoutInflater;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.PopupWindow;
+
+import androidx.core.content.res.ResourcesCompat;
+
+import org.chromium.chrome.R;
+import org.chromium.chrome.browser.rewards.BraveRewardsPanel;
+import org.chromium.chrome.browser.util.TabUtils;
+
+/**
+ * This class is to show success screen
+ * */
+public class RewardsOnBoardingAllSet {
+    private final View mAnchorView;
+    private final PopupWindow mPopupWindow;
+
+    public RewardsOnBoardingAllSet(View anchorView, int deviceWidth) {
+        mAnchorView = anchorView;
+        mPopupWindow = new PopupWindow(anchorView.getContext());
+        mPopupWindow.setHeight(ViewGroup.LayoutParams.WRAP_CONTENT);
+        mPopupWindow.setBackgroundDrawable(ResourcesCompat.getDrawable(
+                anchorView.getContext().getResources(), R.drawable.rewards_panel_background, null));
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
You are right since Brave supports above android-6, so we can remove it. 
but It's ok to keep this condition
https://developer.android.com/reference/android/widget/PopupWindow?hl=en#setElevation(float)